internal/test/resources.go

```diff
@@ -1554,3 +1555,14 @@ func NewConsoleLink() *consolev1.ConsoleLink {
 		},
 	}
 }
+
+func NewApiServer() *configv1.APIServer {
+	return &configv1.APIServer{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "cluster",
+		},
+		Spec: configv1.APIServerSpec{
+			AdditionalCORSAllowedOrigins: []string{},
```

Comment:
I think it would be useful to have a test case that checks the behaviour with another entry present in `AdditionalCORSAllowedOrigins`. This would just ensure that we don't interfere with user-specified values here.

Reply:
I've added an extra entry here and updated the tests to confirm that this extra entry is not affected by the cryostat controller